packages/color-modes/src/custom-properties.ts

```diff
@@ -78,15 +79,24 @@ export const createColorStyles = (theme: Theme = {}) => {
       },
     })(theme)
   }
-  const { colors } = theme
+  const { colors, initialColorModeName, printColorModeName } = theme
   const modes = colors.modes || {}
   const styles = objectToVars('colors', colors)
 
-  Object.keys(modes).forEach((mode) => {
+  Object.keys(modes).forEach(mode => {
     const key = `&.theme-ui-${mode}`
     styles[key] = objectToVars('colors', modes[mode])
   })
 
+  if (printColorModeName) {
+    const mode =
+      printColorModeName === 'initial' ||
+      printColorModeName === initialColorModeName
+        ? colors
+        : modes[printColorModeName]
+    styles['@media (print)'] = objectToVars('colors', mode)
```

Comment:
@lachlanjc shouldn't this be `@media print`? (without parens) I just noticed it doesn't work on Brave/Linux.

Reply:
Yep, you're totally right. I always forget about that since it's the only media query I use without parentheses